browser-ui/server.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+import socketserver
+from http import server # Python 3
+
+PORT = 8000
```

Comment:
```suggestion
from http import server # Python 3
import os

PORT = int(os.getenv("PORT", 8000))
```

Reply:
If we decide to make the port configurable, then we should use argparse instead. For now hard-coded port is good enough.